sale_fixed_discount/README.rst

```diff
@@ -42,52 +42,61 @@ Installation
 ============
 
 This module depends on module 'account_invoice_fixed_discount',
-available in https://github.com/OCA/account-invoicing/tree/16.0/account_invoice_fixed_discount
+available in
+https://github.com/OCA/account-invoicing/tree/16.0/account_invoice_fixed_discount
 
 Configuration
 =============
 
 To configure this module, you need to:
 
-#. Go to *Sales > Configuration > Settings*.
-#. In the *Pricing* section select *Discounts* option to grant discounts on sales order lines.
+1. Go to *Sales > Configuration > Settings*.
+2. In the *Pricing* section select *Discounts* option to grant discounts
+   on sales order lines.
 
 Usage
 =====
 
 To use this module, you need to:
 
-#. Go to *Sales*.
-#. Create a Sales Order and specify the type of discount and fixed/percent
-   discount in a line.
+1. Go to *Sales*.
+2. Create a Sales Order and specify the type of discount and
+   fixed/percent discount in a line.
 
 Bug Tracker
 ===========
 
 Bugs are tracked on `GitHub Issues <https://github.com/OCA/sale-workflow/issues>`_.
 In case of trouble, please check there if your issue has already been reported.
 If you spotted it first, help us to smash it by providing a detailed and welcomed
-`feedback <https://github.com/OCA/sale-workflow/issues/new?body=module:%20sale_fixed_discount%0Aversion:%2016.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+`feedback <https://github.com/OCA/sale-workflow/issues/new?body=module:%20sale_fixed_discount%0Aversion:%2017.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
 
 Do not contact contributors directly about support or help with technical issues.
 
 Credits
 =======
 
 Authors
-~~~~~~~
+-------
 
 * ForgeFlow
 
 Contributors
-~~~~~~~~~~~~
+------------
 
-* Lois Rilo <lois.rilo@forgeflow.com> (www.forgeflow.com)
-* Jordi Ballester <jordi.ballester@forgeflow.com> (www.forgeflow.com)
-* Pieter Paulussen <pieterpaulussen@code-source.be> (www.code-source.be)
+-  Lois Rilo <lois.rilo@forgeflow.com>
+   (`www.forgeflow.com <http://www.forgeflow.com>`__)
```

Comment:
change to https

Reply:
Does pre-commit give you any error?

Reply:
Hi @JordiBForgeFlow 
Generate addons README files from fragments..............................Failed
- hook id: oca-gen-addon-readme
- files were modified by this hook

is this, and it make change 'https' to 'http' by it self, still should I proceed with https ?

Reply:
Leave it for now and attend to my other comments, please.